docs/commands/csv2po.rst

```diff
@@ -88,34 +92,45 @@ The resultant CSV file has the following layout
 |  C     | Target Language | The msgstr or target language               |
 +--------+-----------------+---------------------------------------------+
 
+
 .. _csv2po#examples:
 
 Examples
 ========
 
-These examples demonstrate the use of csv2po::
+These examples demonstrate the use of csv2po:
+
+.. code-block:: console
 
   po2csv -P pot csv
 
-We use the :opt:`-P` option to recognise POT files found in *pot* and convert
-them to CSV files placed in *csv*::
+
+We use the :option:`-P` option to recognise POT files found in :file:`pot` and
```

Comment:
not pinning the upstream apps causes ongoing problems. Often devs want to get on with working on their code not worry about what has changed in upstream versions. When things do break - the easiest fix is to pin it to the known working version.

If the known working upstream has no known security issues *this is a perfectly valid strategy*

I am not sure i follow what you are saying about commits - i figure it would have been a lot clearer if you could have provided a link or some kind of reference.